src/chart/SunburstChart.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { scaleLinear } from 'victory-vendor/d3-scale';
+import { Surface } from '../container/Surface';
+import { Layer } from '../container/Layer';
+import { Sector } from '../shape/Sector';
+import { Text } from '../component/Text';
+
+export interface SunburstData {
+  name: string;
+  value?: number;
+  fill?: string;
+  children?: SunburstData[];
+}
+
+interface TextOptions {
+  fontFamily?: string;
+  fontWeight?: string;
+  paintOrder?: string;
+  stroke?: string;
+  fill?: string;
+}
+
+export interface SunburstChartProps {
+  data?: SunburstData;
+  width?: number;
+  height?: number;
+  padding?: number;
+  ringPadding?: number;
+  innerRadius?: number;
+  children?: React.ReactNode;
+  fill?: string;
+  stroke?: string;
+  textOptions?: TextOptions;
+}
+
+interface DrawArcOptions {
+  r: number;
+  innerR: number;
+  initialAngle: number;
+  childColor?: string;
+}
+
+const defaultTextProps = {
```

Comment:
Are these defaults also used in other places?

Reply:
I couldn't find an example of this anywhere, but seems nice to provide some default styling that stands out better on the chart background than the un-styled text.  I can remove if needed though

Reply:
I agree with you and like this, but wonder whether we should tackle such an improvement separately, and instead focus on consistency for this new chart.